pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: number_to_words
 description: Convert number into words by localeID
-version: 0.0.3
+version: 0.1.0
```

Comment:
This would be an API breaking change full null safety.  Using Semvar, it should roll the major verison.